### ISSUE TITLE
Add ability to report outages

### ISSUE THREAD

**Issue by NeoScaler** [johan12345/EVMap#87]
I would like to see an ability to report outages or errors. 

Currently you have to edit the charger and clicking "Back to entry" on the GoingElectric website. There you can report a problem with the charger.

**Comment by johan12345** [comment]
Yeah, I agree this would be useful! While this feature is part of the JSON API under `https://www.goingelectric.de/stromtankstellen/api.php` that is used on the GoingElectric.de website, it is not part of the [publically documented API](https://www.goingelectric.de/stromtankstellen/api/docs/), so I'm afraid that it's probably not possible to support this feature in the app without breaking the [terms of use](https://www.goingelectric.de/stromtankstellen/api/terms/):
> Es dürfen ausschließlich die über die Schnittstelle bereitgestellten Daten verwendet werden.
Or do you have another idea how to solve this?

There is already a button in the app to go directly to the charger page on GoingElectric (at the very bottom of the details screen), so the additional "Back to entry" step is not necessary.

**Comment by johan12345** [comment]
See also #41 for a similar case.

**Comment by johan12345** [comment]
While this is not possible for GoingElectric, it would be possible with the OpenChargeMap data source, which has been added in #96/#81.
